### DISCUSSION TITLE
Admin with remote_user_guard

### DISCUSSION THREAD

**Discussion by derfloDev** [firefly-iii/firefly-iii#6145] [Firefly III]
Hi, im running FireFlyII with docker-compose and AUTHENTICATION_GUARD=remote_user_guard.
My Authentication Guard is Authelia.
Every thing is working fine so far. I can login an do stuff.
But how do i access the admin Page.
Or how do i tell FireFly which user is the Admin when using remote_user_guard?

**Comment by JC5** [comment]
The first user is the admin. Is there a user registered before you set up remote_user_guard?

**Reply by derfloDev** [reply]
No, the first time I started was with AUTHENTICATION_GUARD=remote_user_guard. So there is no other user than the one from Authelia

**Reply by JC5** [reply]
Thanks! That's a bug! I'll pick it up 👍🏻

**Reply by nottheend** [reply]
I also started firefly-iii the first time with `AUTHENTICATION_GUARD=remote_user_guard`
Unfortunately I deleted the first user and using another one. Is there a way to change the admin? E.g. by setting a flag in the DB or sth

**Comment by viharm** [comment]
@derfloDev , sorry about the slight off topic comment, but are you able to post your _Authelia_, _Nginx_ & _Firefly II_ configs to make it work please?
